Lab_3.html

```diff
@@ -0,0 +1,84 @@
+<br>
+<!-- 
+##### SOURCE BEGIN #####
+%% 
+% *Problem1*
+% 
+% *A1*  I randomly create 100K points from a Gaussian distribution with the 
+% mean value of 12 and the standard variation value of 0.4 in MATLAB. Then I add 
+% bad points of [10., 10.3, 2.1, 0., 0., 15.6, 22.3, 12.7]. The codes and the 
+% boxplot are following:
+
+Gauss_Data=normrnd(12,0.4,[1 100000])
+Integrated_Data = [Gauss_Data, [10., 10.3, 2.1, 0., 0., 15.6, 22.3, 12.7]]
+sboxplot(Integrated_Data)
+%% 
+% 
+% 
+% *A2*  According to the central limit theorem, the mean of the sample also 
+% conforms to the normal distribution. The possibility bad values should smaller 
+% than -3*sigma or bigger than 3*sigma is 99.74%.
+% 
+% *A3*  $Bad values < \mu -3\times sigma\\Bad values>\mu +3\times sigma \\sigma=\sqrt{\frac{1}{n}{\sum_{k=1}^n(x_i-\bar{x})^2}}\]]$
+% 
+% *A4* 
+
+threshold1 = mean(Integrated_Data) -3* std(Integrated_Data)
+threshold2 = mean(Integrated_Data) +3* std(Integrated_Data)
+%% 
+% Therefore, we can find 10., 10.3, 2.1, 0., 0., 15.6, 22.3 values as bad values. 
+% However we cannnot find 12.7.  The TP is 100k, FN is 7, TN is1, FP is 0.
+% 
+% *B* I didn't omit any good values.
+% 
+% *C* There is one commision. I think if we use 4 times of sigma then we can 
+% avoid this mistake.
+% 
+% 
+% 
+% *Problem2*
+% 
+% 
+% 
+% *Problem statement*: we assume movements introduced by subject elements such 
+% as instrument move. As observation points are more than 100, we also assume 
+% this is a norm distribution. 5 sigma means if there is a value out from 5 sigma 
+% interval there is very small possibility introduced by environment, which means 
+% onece we find this outsider that would be the arcsecond we are looking for.
+% 
+% *Mathematical Speaking*: Confidence region is :$(\bar X - \mu)' S'(\bar X 
+% - \mu) < \frac{p}{n-p}F_{1-\alpha;p,n-p}$
+% 
+% 
+% 
+% *Problem3*
+% 
+% *1.* If there is no gamma-ray (null hypothesis), then we expect the background 
+% to be Poissson distributed with a mean of 7200. So what is the possibility of 
+% the background producing events greater than 6800 and less than 7200?
+% 
+% 2. 
+
+background = 7200;
+pois = makedist('Poisson','lambda',background);
+prob = cdf(pois,7200)-cdf(pois,6800);
+prob = 0.5031
+icdf('Normal',1- t,0,1)
```

Comment:
line 75) I think 't' should be changed to 'prob' because when I ran the code on matlab, it says "Undefined function or variable 't'". But changing 't' to 'prob' didn't change the result (-0.0078) when I tested it, so you don't have to worry about the result changing.